modules/ROOT/pages/import-api-specification-exchange.adoc

```diff
@@ -57,6 +57,15 @@ Manage the API specifications linked to Exchange in your project from the API sp
 [NOTE]
 If you want Studio to list your API when searching in Exchange, you must mark your API specification published as an asset as *stable*.
 
+== Rescaffold an API Specification in a Mule Project
+
+After changing your API spec, such as adding a new endpoint, re-scaffold the spec to update your Mule project.
+
+. Right-click on the file.xml of your Mule project.
```

Comment:
Is file.xml the actual name of the file? If so, use monospace.

Reply:
The file name depends on the project name. If the project is called "MyFirstMuleProject" the file will be "myfirstmuleproject.xml". Let's see if my rewording works 🤞